samples/bluetooth/broadcast_audio_sink/prj.conf

```diff
@@ -14,3 +14,5 @@ CONFIG_BT_BUF_ACL_RX_SIZE=255
 CONFIG_BT_BUF_ACL_TX_SIZE=251
 
 CONFIG_BT_DEVICE_NAME="Broadcast Audio Sink"
+
+CONFIG_BT_TINYCRYPT_ECC=y
```

Comment:
For the unicast audio samples, this is only being enabled in the `nrf5340bsim_nrf5340_cpuapp.conf` board file, should we do the same here?